www/includes/easyparliament/metadata.php

```diff
@@ -732,7 +732,7 @@
         'url'			=> 'search/',
         'robots'		=> 'noindex, nofollow',
         'heading'		=> '',
-        'session_vars'	=> array ('q', 's', 'pid', 'o', 'pop')
+        'session_vars'	=> array ('q', 's', 'pid', 'o', 'pop', 'person', 'from', 'to', 'section', 'column')
```

Comment:
This confuses the links. The "Sort by" links have already incorporated all the advanced search terms into their `q` parameter, and now have duplicate advanced search parameters on top. The pagination links, previously having nothing, now also duplicate the parameters in `q` (due to the previous commit change to searchstring) and their own params.

If you don't have the second commit of this PR, only the first, then that fixes it in that all the data is passed through in the pagination links, but within the main query string, not as advanced params (so they then appear in the main box, not in the advanced search boxes). That would fix the bug, at least. If we want the data to remain in the advanced search boxes if it's entered there, then we'll need to also change the code to not include those terms within the query string when it's going to be output along with them in a URL (so only have one compiled query string for the actual searching, not re-output).